### ISSUE TITLE
Is it possible to syntax highlight "not", "and", and "or"

### ISSUE THREAD

**Issue by brevzin** [microsoft/vscode-cpptools#5483]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Windows 10 v1909
- VS Code Version: 1.44.2
- C/C++ Extension Version: 0.26.3
- Other extensions you installed (and if the issue persists after disabling them):
- Does this issue involve using SSH remote to run the extension on a remote machine?: I'm running SSH remote, but it doesn't matter
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

In C++, we have a few keywords-but-not-really-keywords. The most commonly used ones are `and`, `or,` and `not` - which are treated exactly as the tokens `&&`, `||`, and `!`. None of those three (or any of the [rest of them](http://eel.is/c++draft/lex.key#tab:lex.key.digraph)) are syntax highlighted. It would be really great if they were treated as proper keywords and highlighted accordingly.

**Steps to reproduce**

Basically, open a C++ file and type:
```cpp
if (x and y or z and not w)
```
The `if` gets highlighted (purple based on my color scheme), the rest do not at all. In markdown, they _do_ get highlighted red. 

**Expected behavior**

Having `and`, `or`, and `not` in the above fragment also be highlighted. 

<details>
  <summary><strong>Logs</strong></summary>

```
Insert logs here.
```
</details>

**Screenshots**

**Additional context**

**Comment by sean-mcmanus** [comment]
Our extension doesn't provide lexical colorization for keywords/operators or theme colors. Using the "Inspect Editor And Scopes" command, the scoping seems correct, but your theme isn't handling it in the way you desire, but it should be customizable.

![image](https://user-images.githubusercontent.com/19859882/81855490-45bb2a80-9514-11ea-94d6-bc53702bee98.png)

You can use settings:
```
    "editor.tokenColorCustomizations": {
        "textMateRules": [
            {
                "scope": "keyword.operator.and.cpp",
                "settings": {
                    "foreground": "#C586C0"
                }
            },
            {
                "scope": "keyword.operator.or.cpp",
                "settings": {
                    "foreground": "#C586C0"
                }
            },
            {
                "scope": "keyword.operator.not.cpp",
                "settings": {
                    "foreground": "#C586C0"
                }
            }
        ]
    }
```

to get
![image](https://user-images.githubusercontent.com/19859882/81856215-59b35c00-9515-11ea-8259-7a8992546b16.png)

If you would like a theme to have different default colors, you could file an issue on the specific theme.

**Comment by brevzin** [comment]
Thanks! Where do I go to file an issue for that them? I'm using `Dark+ (default dark)`

**Comment by sean-mcmanus** [comment]
@BRevzin https://github.com/microsoft/vscode/issues appears to be the place to report default theme issues.
